packages/services/src/parsing.ts

```diff
@@ -24,53 +28,21 @@ export interface IDictionary<T> {
 /**
  * A utilities class for parsing notebook files.
  */
-export class NotebookParser {
+export class FileParser {
```

Comment:
Minor, should this actually be renamed to `ContentParser` to match the backend?

Reply:
That would probably make sense! I'll change here and in all the relevant frontend `import` statements